packages/gestalt/src/Icon.js

```diff
@@ -37,7 +37,7 @@ type Props = {|
 // $FlowFixMe[signature-verification-failure] flow 0.135.0 upgrade
 const IconNames = Object.keys(icons);
 
-const flipOnRtlIconNames = [
+export const flipOnRtlIconNames = [
```

Comment:
I don't see this used anywhere else in this PR…does this need to be exported?

Reply:
this is legacy from an earlier exploration.

DONE. good catch